### DISCUSSION TITLE
Mpich + Minecraft server?

### DISCUSSION THREAD

**Discussion by Aerospace911** [pmodels/mpich#6583] [Q&A]
Can I run a minecraft server on this and if so, how would i distribute the computing tasks to multiple computers with the goal of creating a powerful minecraft server.

Could I form them into one computer with an OS ? IE: checking how much ram i have would show the combined ram among the computers.

**Comment by hzhou** [comment]
You can easily launch multiple minecraft servers with MPI, but each process of the server need coordinate in order to work together rather than work independently. Thus, you will need to hack on the server code quite significantly. It is quite an interesting idea but it will also be a very involved project.
